### ISSUE TITLE
stuck - need some help 

### ISSUE THREAD

**Issue by pinn73** [OmerTu/GoogleHomeKodi#273]
Not an issue as such, just user error.

I have googlehomekodi installed via docker and this is my setup. I have forwarded port 8089.

```
sudo docker run \
   --detach \
   --publish 8089:8089 \
   --restart always \
   -e KODI_PROTOCOL="http" \
   -e KODI_IP="192.168.0.10" \
   -e KODI_PORT="8088" \
   -e KODI_USER="whatIwrote" \
   -e KODI_PASSWORD="whatIwrote" \
   -e AUTH_TOKEN="whatIwrote" \
   --name googlehomekodi \
   omertu/googlehomekodi
```

This all seems to start okay:
```
CONTAINER ID        IMAGE                   COMMAND                  CREATED                                                                                                     STATUS              PORTS                              NAMES
9df629886bcc        omertu/googlehomekodi   "/sbin/tini -- node …"   9 minutes a                                                                                        go       Up About a minute   0.0.0.0:8089->8089/tcp, 8099/tcp   googlehomekodi
```

But from my limited understanding, I should be able to access the server through a browser at http://192.168.0.10:8089 which I can't. 

ANy help would be greatly appreciated. Ta

**Comment by keydon** [comment]
You are running the docker container on the same host as kodi? then this looks fine.
Leave out the `--detach \` for debugging puposes. Check the output then for errors
Are you sure about port 8089 (default is 8099)? You are publishing 8089, but this does not mean that port 8089 is actually listend to! To make node listen to 8089 you need to tell node that with `-e PORT=8089 \`

**Comment by pinn73** [comment]
hi,

Yes, running both on same machine. 8089 is my goooglehomekodiport and 8088 my kodi one.
When you say 'check the output for errors' , do you mean on IFTT?
Also, unsure what you mean by your last sentence.

Thanks

**Comment by pinn73** [comment]
Ok. Got there in the end. Changed port back to default as it seemed to be set up oddly

`0.0.0.0:8089->8089/tcp, 8099/tcp`

Also, was unsure whether of IFTT access token ,I had to add the one I had created or not as in the tutorial this is unclear  (step b.3.3)

I hadn't read the notes properly but when self-hosting, I should be entering my external ip plus port as the server address in IFTT.

Thanks and hopefully my troubles might help another user! 👍
